src/components/KCheckbox/KCheckbox.vue

```diff
@@ -59,4 +84,63 @@ export default defineComponent({
 @import '@/styles/variables';
 @import '@/styles/functions';
 
+.k-checkbox-label {
+  /* Body/Popover Content - Regular - 400 */
```

Comment:
nit: can we remove a lot of these comments that came from the Figma if they aren't really needed?

Reply:
Removed all the Figma comments, had kept them for reference